ResearchKit/Common/ORKTaskViewController.m

```diff
@@ -298,11 +298,12 @@ - (instancetype)initWithTask:(id<ORKTask>)task taskRunUUID:(NSUUID *)taskRunUUID
     return [self commonInitWithTask:task taskRunUUID:taskRunUUID];
 }
 
-- (instancetype)initWithTask:(id<ORKTask>)task restorationData:(NSData *)data {
+- (instancetype)initWithTask:(id<ORKTask>)task restorationData:(NSData *)data delegate:(nonnull id<ORKTaskViewControllerDelegate>)delegate {
```

Comment:
Also, normally the implementation method signature doesn't have *nullability specifiers*.